File: year-2/programming-design/java/MotorcycleContestClientServer/MotorcycleCommon/src/main/java/dtos/CredentialsDto.java
package dtos;

import java.io.Serializable;

public class CredentialsDto implements Serializable {
    private static final long serialVersionUID = 7331115341259248461L;
    private String name, teamName;
    private int capacity;

    public CredentialsDto(String name, String teamName, int capacity) {
        this.name = name;
        this.teamName = teamName;
        this.capacity = capacity;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getTeamName() {
        return teamName;
    }

    public void setTeamName(String teamName) {
        this.teamName = teamName;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }
}

File: year2/mpp/c#/MotorcycleGrpcServer/MotorcycleCommon/Dtos/CredentialsDto.cs
using System;

namespace MotorcycleContest.Domain.Dtos
{
    [Serializable]
    public class CredentialsDto
    {
        public String Name { get; set; }
        public String TeamName { get; set; }
        public int Capacity { get; set; }

        public CredentialsDto(String name, String teamName, int capacity)
        {
            Name = name;
            TeamName = teamName;
            Capacity = capacity;
        }
        
        public override string ToString()
        {
            return "Name: " + Name + ", Team Name: " + TeamName + ", Capacity: " + Capacity;
        }
    }
}
